resources/backend/js/tabs.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * Provides functionality for tabbed interfaces.
+ *
+ * @since   2.2.5
+ *
+ * @package ConvertKit
+ * @author ConvertKit
+ */
+
+/**
+ * Initializes tabbed interfaces
+ *
+ * @since 	2.2.5
+ */
+function convertKitTabsInit() {
+
+	( function( $ ) {
+
+		// Safely call this function by destroying any previously initialized instances.
+		convertKitTabsDestroy();
+
+		// Iterate through all JS tab instances, initializing each one.
+		$( '.convertkit-js-tabs' ).each(
+			function() {
+
+				var nav_tab_container    = $( this ),
```

Comment:
It's generally better to use `const` (or `let`, but since these stay the same, `const` should work).

Reply:
You have some other usages of var, so same comment for those.